lib/tnl-neigh-cache.c

```
@@ -310,11 +438,34 @@ tnl_neigh_cache_run(void)
 
     ovs_mutex_lock(&mutex);
     CMAP_FOR_EACH(neigh, cmap_node, &table) {
-        if (tnl_neigh_expired(neigh)) {
+        enum tnl_neigh_state state = tnl_neigh_get_state(neigh);
+
+        if (state == TNL_NEIGH_REACHABLE &&
+            tnl_neigh_deadline_expired(&neigh->expires)) {
```

Cppcheck (2.21.0)

[warning] Uninitialized variable

(uninitvar)